proto/ignition/msgs/material.proto

```diff
@@ -114,4 +114,6 @@ message Material
 
   /// \brief Physically Based Rendering (PBR) material properties
   PBR pbr                = 10;
+
+  int32 render_order       = 11;
```

Comment:
Adding a new field breaks ABI, so it can't be done in a released version. This will need to target `main`.